### ISSUE TITLE
Graphical representation of the Domain

### ISSUE THREAD

**Issue by Mehdi-Ali** [ptrefall/fluid-hierarchical-task-network#19]
**Is your feature request related to a problem? Please describe.**
It's hard to visualize when it gets longer.

**Describe the solution you'd like**
a simple tool that converts the domain from the c# syntax to a graphical diagram.

**Comment by ptrefall** [comment]
I agree that a feature like this would greatly aid production of domains. I'll have a look at some options again (it's been a little while since I last evaluated this).
